openapi/SwarmCommon.yaml

```diff
@@ -138,7 +138,7 @@ components:
     DateTime:
       type: string
       format: date-time
-      pattern: '^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{7}\+\d{2}:\d{2})$'
+      #pattern: '^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{7}\+\d{2}:\d{2})$'
```

Comment:
not sure if this was intentional. uncomment?

Reply:
yes, intentional, the format and pattern used together caused a problem with latest openapi generator

Reply:
so maybe we can just keep the old version?

Reply:
No we can't, our spec becomes useless with latest openapi generator if we do

Reply:
i meant not to increment the open api version to `3.0.3`. the generator should then just use the old version. and then we don't need to comment this out. are we using any specific `3.0.3` spec features?

Reply:
ah the specification version increase was unrelated, this broke with the new version of the openapi generator (5.0.0) which must have been recently released, and this ecosystem is full of bugs and inconsistencies, but it's worth to keep checking against latest version of this tool as earlier versions become obsolete anyway